habitat/tasks/rearrange/sub_tasks/articulated_object_sensors.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+from gym import spaces
+
+from habitat.core.embodied_task import Measure
+from habitat.core.registry import registry
+from habitat.core.simulator import Sensor, SensorTypes
+from habitat.tasks.rearrange.rearrange_sensors import RearrangeReward
+
+
+@registry.register_sensor
+class MarkerRelPosSensor(Sensor):
```

Comment:
Providing a set of generic ArticulatedObject sensors is great. These should all be fully documented and typed.